libs/dh/metering-point/feature-measurements/src/components/month.component.ts

```diff
@@ -145,14 +145,21 @@ export class DhMeasurementsMonthComponent {
   private route = inject(ActivatedRoute);
   private fb = inject(NonNullableFormBuilder);
   private transloco = inject(TranslocoService);
-  private sum = computed(() =>
-    this.formatNumber(
-      this.measurements()
-        .map((x) => x.quantity)
-        .filter((quantity) => quantity !== null && quantity !== undefined)
-        .reduce((acc, quantity) => acc + Number(quantity), 0)
-    )
+  private sum = computed(
+    () =>
+      `${this.formatNumber(
+        this.measurements()
+          .map((x) => x.quantity)
+          .filter((quantity) => quantity !== null && quantity !== undefined)
+          .reduce((acc, quantity) => acc + Number(quantity), 0)
+      )} ${this.unit()}`
   );
+  private unit = computed(() => {
```

Comment:
Suggestion: We could add Unit to apps/dh/api-dh/source/DataHub.WebApi/Modules/ElectricityMarket/Types/MeasurementPositionDtoType.cs but its up to you.

Reply:
Yes, but that will only have an effect on the day view. The month and year views use different types 🤔